ansible/shell_plugins/docker.py

```diff
@@ -24,7 +24,7 @@ def join_path(self, *args):
         ## We observe the interactions between ShellModule and ActionModule, and
         ## find the temporary directories Ansible created on remote machine. So we
         ## collect them and copied to docker container in build_module_command
-        if len(args) >= 2 and args[0].startswith('/home/') and args[1] == '':
+        if len(args) >= 2 and args[1] == '':
```

Comment:
Instead of removing it, could you find the generic "home" directory and keep the condition?